regression/contracts/invar_havoc_dynamic_multi-dim_array/test.desc

```diff
@@ -18,4 +18,9 @@ to pass -- we should not mistakenly havoc the allocations, just their values.
 However, the `data[1][2][3] == 0` assertion is expected to fail since `data`
 is havoced.
 
-Blocked on issue #6021 -- alias analysis is imprecise and returns `unknown`.
+Blocked on #6326:
```

Comment:
Is this a PR or a GitHub issue? I'd just add the link here.

Reply:
GitHub numbers both issues and PRs on the same number line. They also have a uniform URL for both: `<repo>/issues/<id>`.

So #6326 is a PR, but the URL [`https://github.com/diffblue/cbmc/issues/6326`](https://github.com/diffblue/cbmc/issues/6326) would work just fine.